src/Util/Strings.php

```diff
@@ -532,7 +532,7 @@ function ($matches) use ($blocks) {
 	}
 
 	/**
-	 * This function converts a PHP's shorhand notation string for file sizes in to an integer number of total bytes.
+	 * This function converts a PHP's shorthand notation string for file sizes into an integer number of total bytes.
```

Comment:
`in to`/`into` has been quite controversial, but I think this change is correct...

Reply:
> This function converts a file size string written in PHP's shorthand notation to an integer number of total bytes.